README.md

```diff
@@ -1,75 +1,80 @@
-* compare Notebooks side-by-side, with diffs for source, markdown, output, and execution count cells
-* share Notebook checkpoints or diffs with persistent links
-* download Notebook checkpoints directly from the UI or Jupyter 
+## What will you get with this integration?
 
-![image](https://docs.neptune.ai/img/app/notebook_comparison.png)
-*Side-by-side comparison of Notebooks.*
+* Log and display notebook checkpoints either manually or automatically during model training
+* Connect notebook checkpoints with model training runs in Neptune
+* Organize checkpoints with names and descriptions 
+* Browse checkpoints history across all notebooks in the project
+* Compare Notebooks side-by-side, with diffs for source, markdown, output, and execution count cells
+* Share Notebook checkpoints or diffs with persistent links
+* Download Notebook checkpoints directly from Neptune or Jupyter 
 
-Technically Neptune + Jupyter integration is implemented as a neptune-notebooks Python package which includes:
-* Standard Jupyter extension - The extension is written in Javascript and is visible in the web-based Jupyter UI. 
-* JupyterLab extension - The extension is written in Javascript and is visible in the web-based Jupyter UI.
-* A very basic CLI (command line interface) for uploading notebooks.
+![Jupyter Notebook rendered in Neptune](https://docs.neptune.ai/img/app/notebook_view.png)
 
 ## Resources
 
 * [Documentation](https://docs.neptune.ai/tools/jupyter/overview/)
-* [Example notebook in the Neptune app](https://app.neptune.ai/neptune-ai/credit-default-prediction/n/exploring-application-table-ac75c237-1630-4109-b532-dd125badec0e/ca1df3be-b2e4-4b26-99d6-b7e98a3d4273)
+* [Example notebook in Neptune](https://app.neptune.ai/neptune-ai/credit-default-prediction/n/exploring-application-table-ac75c237-1630-4109-b532-dd125badec0e/ca1df3be-b2e4-4b26-99d6-b7e98a3d4273)
 
-## Example
+## Setup
 
-```python
-# On the command line, install the neptune-notebooks extension
-pip install neptune-notebooks
+Install the extension:
 
-# Enable the extension for Jupyter
-jupyter nbextension enable --py neptune-notebooks
+```sh
+pip install neptune-notebooks
+```
 
+Enable the extension for Jupyter:
 
-# Install the Neptune client
-pip install neptune-client
+```sh
+jupyter nbextension enable --py neptune-notebooks
 ```
 
 In your Jupyter Notebook environment, some Neptune items appear in your toolbar.
-1. Click the Neptune icon to connect with Neptune.
-2. Click **Upload** to create a first notebook checkpoint in Neptune.
-3. In the Jupyter notebook, import Neptune and start a run:
+
+1. To connect with Neptune, click the Neptune **Configure** button.
+2. Enter your [Neptune credentials](https://docs.neptune.ai/setup/setting_credentials/#finding-your-credentials).
+3. To register the notebook in Neptune, click **Upload**.
+4. To log metadata from the notebook automatically, in the top navigation menu, click **Activate**.
```

Comment:
Activating doesn't automatically log metadata, it just links the runs created in the notebook to the notebook
![image](https://github.com/neptune-ai/neptune-notebooks/assets/41324509/b8112ed9-ce6a-409a-b276-267498204ebf)


```suggestion
4. To link runs to the notebook automatically, in the top navigation menu, click **Activate**.
```

Reply:
Ahh, but... Wait a minute. What does that _do_, exactly? We used to have this "Used by X runs" for notebooks, right? Is it about that, or something else?

Reply:
ah yes, we removed the "Used by X runs" feature.

I don't think "activating" does anything then. Uploading the initial checkpoint to Neptune takes care of logging the notebook in `source_code/notebook`.

Lemme confirm though